Eastern-Research-Group/mywaterway
# Feature/322 update educators page

## Related Issues:
* https://jira.epa.gov/browse/HMW-322

## Main Changes:
* Updated the content on the educators page.

## Steps To Test:
1. Navigate to http://localhost:3000/educators
2. Verify the text and links match up with those in the word doc on the ticket
